### DISCUSSION TITLE
can you disable the “video paused. continue watching?” prompt?

### DISCUSSION THREAD

**Discussion by agent-berry** [qnblackcat/uYouPlus#1191] [Q&A]
I could have sworn there was an option previously to disable this, and I enabled it for like… a week before YouTube pushed out that update that made the ads come back. I know I enabled it because I use YouTube to play music while I draw and I had a good few days of no interruptions while doing so.

I installed the latest version of uYou+ to circumvent the ads, but now I can’t seem to find the option to disable that prompt for the life of me. Was it removed? Is it an A/B setting I found through these threads and then completely forgot about after? Did I somehow gaslight myself into thinking it existed when it didn’t? Help.

tldr: the “video paused. continue watching?” prompt is Annoying and I want it Gone. Is that possible? If so, how do I do it again? thank you in advance for the help ^^

**Comment by mehargiri** [comment]
Yes, it's definitely possible to disable "video paused. Continue Watching" prompt.

The button to disable that prompt is in the uYou settings. I have included pics that can help you find it. Please ask if you need more clarification. 

Step 1
![image](https://github.com/qnblackcat/uYouPlus/assets/84210815/3e8ea823-069f-4b56-afad-5f9aaa5b0340)

Step 2
![image](https://github.com/qnblackcat/uYouPlus/assets/84210815/4340f72e-3a27-422c-a22c-0046f1d1a227)

Step 3
![image](https://github.com/qnblackcat/uYouPlus/assets/84210815/f7600bb9-f347-4d8d-b3ed-7a9c8142e518)

Step 4
![image](https://github.com/qnblackcat/uYouPlus/assets/84210815/f319d81e-1563-40fc-ab2c-b4da518d1a26)

**Reply by agent-berry** [reply]
Thank you so much!! you’ve saved my drawing binges again 😭💖
